README.md

```diff
@@ -103,51 +55,94 @@ If all nodes in the cluster are running Bottlerocket and require the same `updat
 
-Now we can install the Bottlerocket update operator using the recommended configuration defined [here](./update-operator.yaml):
+### A Note About Removing Labels
+
+Should you decide that the update operator should no longer manage a node, removing the `updater-interface-version` is not quite sufficient to remove the update operator components responsible for that node.
+The update operator associates a Kubernetes Custom Resource with each node. While the Custom Resource will be garbage collected if the node itself is deleted, you must manually clean up the Custom Resource if you choose to delete only the `updater-interface-version`. The Custom Resource can be deleted like so:
 
 ```sh
-kubectl apply -f ./update-operator.yaml
+# Set this to the name of the node you wish to stop managing with the update operator.
+NODE_NAME="my-node-name"
+kubectl delete brs brs-${NODE_NAME} --namespace brupop-bottlerocket-aws 
 ```
 
-## Coordination
+## Operation
 
-The update operator controller and agent processes communicate by updating the node's annotations as the node steps through an update.
-The node's annotations are used to communicate an `intent` which acts as a goal or target that is set by the controller.
-The controller uses internal policy checks to manage which `intent` should be communicated to an agent.
-This allows the controller to fully own and coordinate each step taken by agents throughout its cluster.
-No agent process will otherwise take any disruptive or intrusive action without being directed by the controller to do so (in fact the agent is limited to periodic metadata updates *only*).
+### Overview
 
-To handle and respond to `intent`s, the agent and controller processes subscribe to Kubernetes' node resource update events.
-These events are emitted whenever update is made on the subscribed to resource, including: heartbeats, other node status changes (pods, container image listing), and metadata changes (labels and annotations).
+The update operator controller and agent processes communicate using Kubernetes [Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/), with one being created for each node managed by the operator.
+The Custom Resource created by the update operator is called a BottlerocketShadow resource, or otherwise shortened to `brs`.
+The Custom Resource's Spec is configured by the controller to indicate a desired state, which guides the agent components.
+The update operator's agent component keeps the Custom Resource Status updated with the current state of the node.
+More about Spec and Status can be found in the [Kubernetes documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#object-spec-and-status).
 
+Additionally, the update operator's controller and apiserver components expose metrics which can be configured to be collected by Prometheus.
```

Comment:
nit: Maybe link to the section below? "#### Monitoring Cluster History and Metrics with Prometheus"